specification/indices/get_data_stream_settings/IndicesGetDataStreamSettingsRequest.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { RequestBase } from '@_types/Base'
+import { Indices } from '@_types/common'
+import { Duration } from '@_types/Time'
+
+/**
+ * Get data stream settings.
+ * Get setting information for one or more data streams.
+ * @rest_spec_name indices.get_data_stream_settings
+ * @availability stack stability=stable visibility=feature_flag feature_flag=logs_stream
+ * @availability serverless stability=stable visibility=feature_flag feature_flag=logs_stream
+ * @index_privileges view_index_metadata
+ * @doc_id indices-get-data-stream-settings
```

Comment:
Per https://github.com/elastic/elasticsearch-specification/blob/main/docs/modeling-guide.md#doc_tag it's generally a good idea to explicitly define the tag that affects which docs group the API belongs to. For example, assuming you want it to belong in https://www.elastic.co/docs/api/doc/elasticsearch/group/endpoint-data-stream, it should use the "data stream" tag:
```suggestion
 * @doc_id indices-get-data-stream-settings
 * @doc_tag data stream
```